pressbooks/pressbooks
# feat: make "Import" a first level menu item

Issue #2572 

This PR moves the previous "Tools > Import" to a first level menu item "Import". 

The functionality should behave the same way as before, however as a side effect of this change the Import URL changed from `BOOK_URL/wp-admin/tools.php?page=pb_import` to `BOOK_URL/wp-admin/admin.php?page=pb_import`.

### How to test

1. Create a book and navigate to its dashboard
1. Make sure you see a new "Import" item just below "Export"
1. Make sure you no longer see an "Import" item under "Tools" menu
1. Import content to make sure the functionality behaves as expected
